components/collapsible-panel/collapsible-panel.js

```diff
@@ -112,7 +112,7 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 				border: 1px solid var(--d2l-color-mica);
 				border-radius: 0.4rem;
 			}
-			:host(:not([expanded]):not([skeleton])) .d2l-collapsible-panel {
+			:host(:not([expanded]):not([skeleton-display])) .d2l-collapsible-panel {
```

Comment:
There are a lot of components using `SkeletonMixin` with custom CSS like this outside of core that will need the same update (if they want to work with groups). I can't think of a way around it, but just wanted to point it out.

We'll want to update the documentation accordingly.

Reply:
Cool. Alternate name that comes to mind: `skeleton-active`

Reply:
And at least changing this can be updated gradually (worst case, you un-skeletize early and ignore the rest of the group) versus changing the property name and having to update every consumer to avoid breaking things. Could we reflect until we've updated everyone, then stop reflecting so it's clear no one should rely on the `skeleton` attribute going forward? 🤔 

Can't think of any good names 😬

Reply:
`skeleton-active` it is!